<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PWGenerator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src_img.qrc">
    <normaloff>:/imgs/imgs/envelope.png</normaloff>:/imgs/imgs/envelope.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="container">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(2, 16, 21);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="header">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>17</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="styleSheet">
             <string notr="true">image: url(:/imgs/imgs/website.png);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Open Sans</family>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: qconicalgradient(cx:0, cy:0.131, angle:316.2, stop:0.0625 rgba(254, 123, 109, 255), stop:0.556754 rgba(214, 204, 190, 255), stop:0.960227 rgba(255, 198, 101, 255));</string>
            </property>
            <property name="text">
             <string>Password Generator</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="body">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QFrame" name="frame">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLineEdit" name="website">
               <property name="font">
                <font>
                 <family>Open Sans</family>
                 <pointsize>17</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: white;
border:2px solid  qconicalgradient(cx:0, cy:0.131, angle:316.2, stop:0.0625 rgba(254, 123, 109, 255), stop:0.556754 rgba(214, 204, 190, 255), stop:0.960227 rgba(255, 198, 101, 255));
padding:5px</string>
               </property>
               <property name="placeholderText">
                <string>Web....</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="email">
               <property name="font">
                <font>
                 <family>Open Sans</family>
                 <pointsize>17</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: white;
border:2px solid  qconicalgradient(cx:0, cy:0.131, angle:316.2, stop:0.0625 rgba(254, 123, 109, 255), stop:0.556754 rgba(214, 204, 190, 255), stop:0.960227 rgba(255, 198, 101, 255));
padding:5px</string>
               </property>
               <property name="placeholderText">
                <string>Email....</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="password">
               <property name="font">
                <font>
                 <family>Open Sans</family>
                 <pointsize>17</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: white;
border:2px solid  qconicalgradient(cx:0, cy:0.131, angle:316.2, stop:0.0625 rgba(254, 123, 109, 255), stop:0.556754 rgba(214, 204, 190, 255), stop:0.960227 rgba(255, 198, 101, 255));
padding:5px</string>
               </property>
               <property name="placeholderText">
                <string>Password....</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="generatepass">
               <property name="font">
                <font>
                 <family>Open Sans</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(253, 116, 109);
color:white;
padding:10px 12px;</string>
               </property>
               <property name="text">
                <string>Generate Password</string>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="submitbtn">
               <property name="font">
                <font>
                 <family>Open Sans</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color:rgb(255, 205, 100);
color:rgb(2, 16, 21);
padding:10px 12px;</string>
               </property>
               <property name="text">
                <string>Submit</string>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLineEdit" name="web_search">
               <property name="font">
                <font>
                 <family>Open Sans</family>
                 <pointsize>17</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: white;
border:2px solid  qconicalgradient(cx:0, cy:0.131, angle:316.2, stop:0.0625 rgba(254, 123, 109, 255), stop:0.556754 rgba(214, 204, 190, 255), stop:0.960227 rgba(255, 198, 101, 255));
padding:5px</string>
               </property>
               <property name="placeholderText">
                <string>Web....</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="getinfo">
               <property name="font">
                <font>
                 <family>Open Sans</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(253, 116, 109);
color:white;
padding:10px 12px;</string>
               </property>
               <property name="text">
                <string>Get Info</string>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_5">
               <property name="font">
                <font>
                 <family>Open Sans</family>
                 <pointsize>17</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: white;
border:2px solid  qconicalgradient(cx:0, cy:0.131, angle:316.2, stop:0.0625 rgba(254, 123, 109, 255), stop:0.556754 rgba(214, 204, 190, 255), stop:0.960227 rgba(255, 198, 101, 255));
padding:5px</string>
               </property>
               <property name="placeholderText">
                <string>Email....</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_6">
               <property name="font">
                <font>
                 <family>Open Sans</family>
                 <pointsize>17</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: white;
border:2px solid  qconicalgradient(cx:0, cy:0.131, angle:316.2, stop:0.0625 rgba(254, 123, 109, 255), stop:0.556754 rgba(214, 204, 190, 255), stop:0.960227 rgba(255, 198, 101, 255));
padding:5px</string>
               </property>
               <property name="placeholderText">
                <string>Password....</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Preferred</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="src_img.qrc"/>
 </resources>
 <connections/>
</ui>
